src/index.css

```diff
@@ -119,3 +119,8 @@ p.MuiTypography-h5.MuiTypography-paragraph {
 .new-years.ph-item .ph-row > div > * {
     background-color: #2b353a;
 }
+
+#fireworks > canvas {
+    height: 100% !important;
+    width: 100% !important;
```

Comment:
do you need importants on these?

Reply:
bruh...

Reply:
I'll try to increase the specificity

Reply:
Not having any luck here. Trying to look at the underlying code to figure out why it's not overwriting but I'm not sure what they are doing under the hood to set those styles right now.

Reply:
importants are fine...they are using inline styles to set the width and height based on the container size (but they use fixed pixels and don't adjust to a resized window).